docs/HowTo/Find-and-Connect/Managing-Peers.md

```diff
@@ -2,72 +2,91 @@
 description: Managing Hyperledger Besu peers
 ---
 
-# Managing peers
+# Manage peers
 
-## Limiting peers
+Hyperledger Besu peer-to-peer (P2P) discovery happens periodically based on the number of peers in a network and the
+node's [peer limit](#limit-peers).
 
-Limiting peers reduces the bandwidth used by Hyperledger Besu. Limiting peers also reduces the CPU
+The frequency of discovery isn't configurable, but you can [limit remote connections](#limit-remote-connections) in
+public networks and [randomly prioritize connections](../../Reference/CLI/CLI-Syntax.md#random-peer-priority-enabled)
+in small, stable networks.
+
+!!! info
+
+    You can use [`admin_addPeer`](../../Reference/API-Methods.md#admin_addpeer) to attempt a specific connection, but
+    this isn't P2P discovery.
+
+We recommend [using bootnodes](Bootnodes.md) to initially discover peers.
+
+## Limit peers
+
+Limiting peers reduces the bandwidth used by Besu. Limiting peers also reduces the CPU
 time and disk access used to manage and respond to peers.
 
 To reduce the maximum number of peers, use the
 [`--max-peers`](../../Reference/CLI/CLI-Syntax.md#max-peers) option. The default is 25.
 
-## No discovery
+## Limit remote connections
 
-To disable P2P discovery, use the
-[`--discovery-enabled`](../../Reference/CLI/CLI-Syntax.md#discovery-enabled) option.
+In public networks, enabling the [remote connection limits](../../Reference/CLI/CLI-Syntax.md#remote-connections-limit-enabled)
+is necessary to prevent eclipse attacks, especially when using [`--sync-mode`](../../Reference/CLI/CLI-Syntax.md#sync-mode)
+and [`--fast-sync-min-peers`](../../Reference/CLI/CLI-Syntax.md#fast-sync-min-peers).
 
-With discovery disabled, peers that have already discovered or are otherwise configured to connect
-to the local node (for example, using
-[`admin_addPeer`](../../Reference/API-Methods.md#admin_addpeer)) can open connections.
-[Static nodes](Static-Nodes.md) can also open connections.
+In private and permissioned networks with a level of trust between peers, enabling the remote connection limits is
+unnecessary and might adversely affect the speed at which nodes can join the network.
+Limiting remote connections may cause a clique of nodes to form when the number of nodes in the network is slightly
```

Comment:
Suggestion: "may cause" -> "can cause", it's more than hypothetical to me, we should insist that it can really happen.